example/config/.mocharc.jsonc

```diff
@@ -0,0 +1,14 @@
+// This config file contains Mocha's defaults.
+// As you can see, comments are allowed.
+// This same configuration could be provided in the `mocha` property of your
+// project's `package.json`.
+{
+  diff: true,
```

Comment:
No double quotes are allowed in jsonc?

I didn't know about JSONC.

Reply:
Good point this is a leftover of JSON5 support introduced earlier bur then dropped.
The double quotes are actually required with the JSON parser. I'll fix the example.